test/testBrowserLanguage.js

```diff
@@ -0,0 +1,139 @@
+import browserLanguage from '../src/nginxconfig/util/browser_language';
+
+class MockLocales{
+    static languages = [];
+
+    static language = null;
+
+    static IntlBackup = null;
+
+    static navigator = null;
+
+    static setNavigatorLanguages(langs){
```

Comment:
```suggestion
    static setNavigatorLanguages(langs) {
```
Formatting consistency